FrontEnd/src/components/ProfilePage/FormComponents/DeleteProfileComponent/DeleteProfilePage.js

```diff
@@ -16,9 +16,13 @@ const DeleteProfilePage = (props) => {
     return (
         <div>
             <div className={css['text__for__delete']}>Видалити акаунт</div>
-            <button className={css['button__delete']}  onClick={() => setModal(true)} style={{marginBottom: 400}}>Видалити</button>
+            <button
+                className={css['button__delete']}
+                onClick={() => setModal(true)} style={{ marginBottom: 400 }}>
```

Comment:
Do not use inline styles, move it into css

Reply:
was fixed